### ISSUE TITLE
failures when bringing up docker containers

### ISSUE THREAD

**Issue by wideglide** [fkie-cad/LuckyCAT#1]
I'm getting two errors when bringing up the docker containers, and I'm not sure what's going wrong.  Have you seen either of these errors?

```
frontend_1  | Traceback (most recent call last):
frontend_1  |   File "./luckycat/frontend/luckycat_frontend.py", line 2, in <module>
frontend_1  |     from luckycat.frontend.app import app
frontend_1  |   File "./luckycat/frontend/app.py", line 23, in <module>
frontend_1  |     add_flask_security(app)
frontend_1  |   File "./luckycat/frontend/security/init_security.py", line 53, in add_flask_security
frontend_1  |     create_default_user_and_roles(user_datastore)
frontend_1  |   File "./luckycat/frontend/security/init_security.py", line 33, in create_default_user_and_roles
frontend_1  |     flash('Added default user on first request', 'success')
frontend_1  |   File "/usr/local/lib/python3.5/dist-packages/flask/helpers.py", line 408, in flash
frontend_1  |     flashes = session.get('_flashes', [])
frontend_1  |   File "/usr/local/lib/python3.5/dist-packages/werkzeug/local.py", line 347, in __getattr__
frontend_1  |     return getattr(self._get_current_object(), name)
frontend_1  |   File "/usr/local/lib/python3.5/dist-packages/werkzeug/local.py", line 306, in _get_current_object
frontend_1  |     return self.__local()
frontend_1  |   File "/usr/local/lib/python3.5/dist-packages/flask/globals.py", line 37, in _lookup_req_object
frontend_1  |     raise RuntimeError(_request_ctx_err_msg)
frontend_1  | RuntimeError: Working outside of request context.
frontend_1  |
frontend_1  | This typically means that you attempted to use functionality that needed
frontend_1  | an active HTTP request.  Consult the documentation on testing for
frontend_1  | information about how to avoid this problem.
frontend_1  | unable to load app 0 (mountpoint='') (callable not found or import error)
```

```
backend_1   | Traceback (most recent call last):
backend_1   |   File "/usr/local/bin/luckycat_backend.py", line 4, in <module>
backend_1   |     __import__('pkg_resources').run_script('Lucky-CAT==0.1', 'luckycat_backend.py')
backend_1   |   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 719, in run_script
backend_1   |     self.require(requires)[0].run_script(script_name, ns)
backend_1   |   File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1504, in run_script
backend_1   |     exec(code, namespace, namespace)
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/Lucky_CAT-0.1-py3.5.egg/EGG-INFO/scripts/luckycat_backend.py", line 33, in <module>
backend_1   |     main()
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/Lucky_CAT-0.1-py3.5.egg/EGG-INFO/scripts/luckycat_backend.py", line 11, in main
backend_1   |     crash_verification_sender = CrashVerificationSender.CrashVerificationSender()
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/Lucky_CAT-0.1-py3.5.egg/luckycat/backend/CrashVerificationSender.py", line 22, in __init__
backend_1   |     if not self.wq.queue_exists(self.ver_queue):
backend_1   |   File "/usr/local/lib/python3.5/dist-packages/Lucky_CAT-0.1-py3.5.egg/luckycat/backend/WorkQueue.py", line 18, in queue_exists
backend_1   |     conn = pika.BlockingConnection(pika.ConnectionParameters(self._host))
backend_1   |   File "/usr/lib/python3/dist-packages/pika/adapters/blocking_connection.py", line 339, in __init__
backend_1   |     self._process_io_for_connection_setup()
backend_1   |   File "/usr/lib/python3/dist-packages/pika/adapters/blocking_connection.py", line 374, in _process_io_for_connection_setup
backend_1   |     self._open_error_result.is_ready)
backend_1   |   File "/usr/lib/python3/dist-packages/pika/adapters/blocking_connection.py", line 395, in _flush_output
backend_1   |     raise exceptions.ConnectionClosed()
backend_1   | pika.exceptions.ConnectionClosed
luckycat_backend_1 exited with code 1
```

**Comment by wideglide** [comment]
the first error seems to be due to refactoring and no app_context here:
https://github.com/fkie-cad/LuckyCAT/blob/5331741a6a9a4624d175be7d2ac478f798d5770c/luckycat/frontend/security/init_security.py#L33

**Comment by 0roman** [comment]
The first error was fixed in 40544f0397cda6d1922cd3f4d81d65b8dad63427. The second error occurs, because the backend starts too fast on your system. The queue is not reachable. Increase the sleep time in ```docker/backend/start_backend.sh```, for example to 15.
https://github.com/fkie-cad/LuckyCAT/blob/40544f0397cda6d1922cd3f4d81d65b8dad63427/docker/backend/start_backend.sh#L9
Then restart LuckyCAT.
